### ISSUE TITLE
Bitrate

### ISSUE THREAD

**Issue by bobreese** [jcrossbdn/fpp-after-hours#26]
The fpp logs in V8 show the following errors for Bitrate

2024-08-21T04:16:45.212739-05:00 FPP8test systemd[1]: Starting mpd.service - Music Player Daemon...
2024-08-21T04:16:46.136126-05:00 FPP8test mpd[2127]: config: option 'bitrate' on line 7 was not recognized
2024-08-21T04:16:46.136378-05:00 FPP8test mpd[2127]: config: option 'bitrate' on line 15 was not recognized
2024-08-21T04:16:46.136468-05:00 FPP8test mpd[2127]: config: option 'bitrate' on line 23 was not recognized
2024-08-21T04:16:46.136556-05:00 FPP8test mpd[2127]: config: option 'bitrate' on line 31 was not recognized
2024-08-21T04:16:46.136685-05:00 FPP8test systemd[1]: Started mpd.service - Music Player Daemon.

**Comment by jcrossbdn** [comment]
can you please provide your current fpp os version, osbuild, platform and output of "mpc version" ?

**Comment by bobreese** [comment]
I have the latest and greatest Beta2 of FPP8
Running on a Pi4
Where would I find the mpc version info you are looking for?

**Comment by jcrossbdn** [comment]
from Help/SSH Shell, sign in with fpp as username and falcon as password then type in mpc version and hit enter

**Comment by bobreese** [comment]
Let me know if you need anything else.
mpd version: 0.23.5
![FPP Info](https://github.com/user-attachments/assets/1430d351-cdff-4fb1-bf94-67411b6edba1)

**Comment by jcrossbdn** [comment]
Works properly on 8alpha, not sure what changed in 8beta.  I will have to switch my pi over to that to test.

**Comment by bobreese** [comment]
The plugin Does Work.
Just did not know why the errors were in the logs and wanted to make sure you were aware.

**Comment by jcrossbdn** [comment]
This is interesting.  I was able to replicate the issue.  I checked the man pages and bitrate is still a valid variable but seems to work without it there.  I am hesitant to remove it from the config generator as I'm not sure if that would break some users configurations or not at this point.

Absolutely I do want to know about any logged errors whether they cause issue or not.  I usually keep my pi on the current release and not the beta so I have to play catchup when a new release breaks stuff :)

I think for now we should just monitor this.  The error only appears to log on startup so I don't have concerns that this will muddy up or flood the FPP error logs.

Thank you for reporting!

**Comment by bobreese** [comment]
Where is this config file?BobOn Aug 21, 2024, at 5:22 PM, Jason Cross ***@***.***> wrote:﻿
This is interesting.  I was able to replicate the issue.  I checked the man pages and bitrate is still a valid variable but seems to work without it there.  I am hesitant to remove it from the config generator as I'm not sure if that would break some users configurations or not at this point.
Absolutely I do want to know about any logged errors whether they cause issue or not.  I usually keep my pi on the current release and not the beta so I have to play catchup when a new release breaks stuff :)
I think for now we should just monitor this.  The error only appears to log on startup so I don't have concerns that this will muddy up or flood the FPP error logs.
Thank you for reporting!

**Comment by jcrossbdn** [comment]
The production config is at /etc/mpd.confThere are processes in the fpp-after-hours plug-in that may undo any changes you make to this file, for example a change to the sound card would force a reload.After a manual change you would need to restart the mpd service or reboot the pi for changes to take effect.JasonSent from my iPhoneOn Aug 21, 2024, at 7:38 PM, Bob Reese ***@***.***> wrote:﻿
Where is this config file?BobOn Aug 21, 2024, at 5:22 PM, Jason Cross ***@***.***> wrote:﻿

This is interesting.  I was able to replicate the issue.  I checked the man pages and bitrate is still a valid variable but seems to work without it there.  I am hesitant to remove it from the config generator as I'm not sure if that would break some users configurations or not at this point.

Absolutely I do want to know about any logged errors whether they cause issue or not.  I usually keep my pi on the current release and not the beta so I have to play catchup when a new release breaks stuff :)

I think for now we should just monitor this.  The error only appears to log on startup so I don't have concerns that this will muddy up or flood the FPP error logs.

Thank you for reporting!
